gammapy/data/data_store.py

```diff
@@ -584,9 +585,94 @@ def make_exposure_image(self, fov, energy_range):
         """
         raise NotImplementedError
 
+    def make_psf(self, source_position, energy=None, theta=None):
+        """Evaluating the energydependent psf for a given energy array or theta array
```

Comment:
Suggest: `Make energy-dependent PSF for a given source position.` as description.